Document best practices for implementing ParameterResolvers (#3829)
junit-team/junit5: documentation/src/docs/asciidoc/user-guide/extensions.adoc

`getExecutableInvoker()` method in the `ExtensionContext`.
====

[[extensions-parameter-resolution-conflicts]]
==== Parameter Conflicts

If multiple implementations of `ParameterResolver` that support the same type are

> declared within a test class, a `ParameterResolutionException` will be thrown, with a

Reviewer: ```suggestion
registered for a test, a `ParameterResolutionException` will be thrown, with a
```

Author: Wording updated

> message to indicate that competing types have been discovered. See the following example:

Reviewer: ```suggestion
message to indicate that competing resolvers have been discovered. See the following example:
```

Author: Wording updated
